diff --git a/browser/components/preferences/config/account-sync.mjs b/browser/components/preferences/config/account-sync.mjs
index b503987a08e2ce64bfc01c4e3a21e5e19ef834f0..768b6c83d75ab8a344dc6b348621b8d8f97d2549 100644
--- a/browser/components/preferences/config/account-sync.mjs
+++ b/browser/components/preferences/config/account-sync.mjs
@@ -42,6 +42,7 @@ Preferences.addAll([
   { id: "services.sync.engine.creditcards", type: "bool" },
   { id: "services.sync.engine.addons", type: "bool" },
   { id: "services.sync.engine.prefs", type: "bool" },
+  { id: "services.sync.engine.workspaces", type: "bool" },
 ]);
 
 /**
@@ -547,6 +548,11 @@ const SYNC_ENGINE_SETTINGS = [
   },
   { id: "syncAddons", pref: "services.sync.engine.addons", type: "addons" },
   { id: "syncSettings", pref: "services.sync.engine.prefs", type: "settings" },
+  {
+    id: "syncWorkspaces",
+    pref: "services.sync.engine.workspaces",
+    type: "workspaces",
+  },
 ];
 
 SYNC_ENGINE_SETTINGS.forEach(({ id, pref }) => {
